### ISSUE TITLE
FlexibleColumn component has no accessible name

### ISSUE THREAD

**Issue by guilhermerenna** [SAP/fundamental-ngx#14249]
### Describe your proposal

The expand/collapse buttons rendered inside the FlexibleColumn use only the title attribute to convey their purpose in Fundamental 0.58.x. Screen readers (e.g. VoiceOver on macOS) do not compute title as an accessible name when the button receives focus — they treat it as a tooltip only. As a result, the button has no accessible name and is announced as ? by VoiceOver.

Important note:
in 0.62.3 it works perfectly fine, so 0.58.x can probably use the same fix available in the newest version.

Unfortunately we cannot use the newest library because it's Angular 21, and we're stuck in Angular 20.

Image of the issue reproduced generically:

<img width="1766" height="1553" alt="Image" src="https://github.com/user-attachments/assets/f710c578-0cee-411d-b6fd-805a71428a35" />

### Which versions of Angular and Fundamental Library for Angular are affected?

Angular: 20, Fundamental Library: 0.58.4

### Affected Component

FlexibleColumn

### Steps to Reproduce (for bugs)

1. Open the VoiceOver accessibility feature or equivalent (In MacOS it's under Settings, you can search for VoiceOver and turn it on)
2. Navigate to a page that contains the FlexibleColumn design
3. Tab until you select the button that expands/shrinks the columns
4. Look/listen to what the reader announces

Actual result:
VoiceOver says "?, button, group"

Expected result:
VoiceOver should say "Expand first column, button, group"

### Priority

Low

### Declaration

- [x] I'm not disclosing any internal or sensitive information.

**Comment by droshev** [comment]
@guilhermerenna have you tested it with JAWS? this is the official supported tool? If not, can you try to reproduce it (or not) and come back to us?

**Comment by guilhermerenna** [comment]
Hi deno!
This is the same as the other one: I haven't tested with JAWS myself, but the reps who did the Audit did test used JAWS 2026.2602.49.
As the other one, I cannot copy it directly here since it's internal docs, but it's all available inside the internal Jira CXMC-58864

**Comment by InnaAtanasova** [comment]
hi @guilhermerenna  I added a fix for the issue and downported it. It's available with version 0.58.8. Please check and let us know if the issue is resolved on your side.

**Comment by guilhermerenna** [comment]
Hi Inna,
sorry, I missed your last message! 

Our team was able to upgrade to Angular 21 around 3 weeks ago, so we moved to Fundamental 0.62.x and the FlexibleColumn looks fine. 

But other teams using Angular 20 / Fundamental 0.58.x should still benefit from this fix. Thanks for providing it!
